crates/hir-ty/src/mir/eval/tests.rs

```diff
@@ -87,6 +88,42 @@ fn main() {
     );
 }
 
+#[test]
+fn panic_fmt() {
+    // panic!
+    // -> panic_2021 (builtin macro redirection)
+    //   -> #[lang = "panic_fmt"] core::panicking::panic_fmt (hooked by CTFE for redirection)
+    //   -> core::panicking::const_panic_fmt
+    //     -> #[rustc_const_panic_str] core::panicking::panic_display (hooked by CTFE for builtin panic)
+    //       -> Err(ConstEvalError::Panic)
+    check_pass(
+        r#"
+//- minicore: fmt, panic
```

Comment:
somehow these tests don't work correctly yet, they currently fail MIR building (also, they shouldn't be `check_pass` but `check_panic` or whatever). I am currently [unable to debug the build failure](https://rust-lang.zulipchat.com/#narrow/stream/185405-t-compiler.2Frust-analyzer/topic/print.20path.20of.20FunctionId), so help welcome

Reply:
I dropped this test, will do it in a follow-up PR. I added a simpler test for the hover behavior, which should be enough end-to-end.